internal/provider/resources/work_pool_test.go

```diff
@@ -77,7 +72,8 @@ func TestAccResource_work_pool(t *testing.T) {
 				Config: fixtureAccWorkPoolCreate(workspace.Resource, randomName, poolType, baseJobTemplate, true),
 				Check: resource.ComposeAggregateTestCheckFunc(
 					testAccCheckWorkPoolExists(workPoolResourceName, &workPool),
-					testAccCheckWorkPoolValues(&workPool, &api.WorkPool{Name: randomName, Type: poolType, BaseJobTemplate: baseJobTemplateMap, IsPaused: true}),
+					testAccCheckWorkPoolValues(&workPool, &api.WorkPool{Name: randomName, Type: poolType, IsPaused: true}),
+					resource.TestCheckResourceAttr(workPoolResourceName, "base_job_template", baseJobTemplateExpected),
```

Comment:
Instead of checking the values of the deployment, we can now go one step further and also test that the value stored in the resource via state is what we expect (9ff43f3).